src/templates/admin/elements/core/user_search_list.html

```diff
@@ -0,0 +1,46 @@
+{% load roles %}
+{% user_roles request.journal user as account_roles %}
+
+<table class="small scroll" id='users'>
+    <thead>
+    <tr>
+        <th>Username</th>
+        <th>First Name</th>
+        <th>Last Name</th>
+        {% if settings.DEBUG %}
+            <th>Hijack</th>{% endif %}
+    </tr>
+    </thead>
+    <tr>
+        <td>{{ user.username }}</td>
+        <td>{{ user.first_name }}</td>
+        <td>{{ user.last_name }}</td>
+        {% if settings.DEBUG %}
+            <td>
+                <button type="submit" class="tiny button"
+                        formaction="/hijack/{{ user.id }}/?next={% url 'core_dashboard' %}">
+                    Hijack User
+                </button>
+            </td>{% endif %}
```

Comment:
We should add the email address to the table, since it is used in the form for searches.

One other header that would be usefull here is "is_active" which is one that trips editors a lot too.